apple/RNSVGSvgViewModule.mm

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright (c) 2015-present, Horcrux.
+ * All rights reserved.
+ *
+ * This source code is licensed under the MIT-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#import "RNSVGSvgViewModule.h"
+#import <React/RCTBridge.h>
+#import <React/RCTUIManager.h>
+#import <React/RCTUIManagerUtils.h>
+#import "RNSVGSvgView.h"
+
+@implementation RNSVGSvgViewModule
+
+RCT_EXPORT_MODULE()
+
+@synthesize viewRegistry_DEPRECATED = _viewRegistry_DEPRECATED;
+@synthesize bridge = _bridge;
+
+RCT_EXPORT_VIEW_PROPERTY(bbWidth, RNSVGLength *)
+RCT_EXPORT_VIEW_PROPERTY(bbHeight, RNSVGLength *)
+RCT_EXPORT_VIEW_PROPERTY(minX, CGFloat)
+RCT_EXPORT_VIEW_PROPERTY(minY, CGFloat)
+RCT_EXPORT_VIEW_PROPERTY(vbWidth, CGFloat)
+RCT_EXPORT_VIEW_PROPERTY(vbHeight, CGFloat)
+RCT_EXPORT_VIEW_PROPERTY(align, NSString)
+RCT_EXPORT_VIEW_PROPERTY(meetOrSlice, RNSVGVBMOS)
+RCT_EXPORT_VIEW_PROPERTY(tintColor, UIColor)
+RCT_REMAP_VIEW_PROPERTY(color, tintColor, UIColor)
```

Comment:
@WoLewicki, was this a typo? We don't need these, right?

Exporting ViewProperties on TurboModules doesn't make much sense. So, we should get rid of these macros.